### ISSUE TITLE
Respect case of container name

### ISSUE THREAD

**Issue by notoriousjere** [dokeraj/AndroTainer#45]
Androtainer makes the first letter of each container name capitalized. Could you make it not do that? I prefer lowercase names and the app is changing them unnecessarily.

**Comment by dokeraj** [comment]
It's a visual / styling preference for me, why do you prefer the lowercase style?

**Comment by notoriousjere** [comment]
It's not that I prefer lowercase, I just think the container name shouldn't be modified by the app. You can easily capitalize the first letter of your container names when you're making them. I'd prefer the names look as they do on the portainer web page.

**Comment by hirenshah** [comment]
I agree, the app should show container names as they have been set and not apply any kind of formatting.

Everyone is entitled to name as per their preference, and if you named containers based on your preference then formatting wouldn't be required.
